web_console/adding-user-preferences.adoc

```diff
@@ -6,7 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can change the default preferences for your profile to meet your requirements. You can set your default project, topology view (graph/list), editing medium (form or YAML), and language preferences.
+[role="_abstract"]
+You can change the default preferences for your profile to meet your requirements. You can set your default project, topology view (graph/list), editing medium (form or YAML), language preferences, and resource type.
```

Comment:
```suggestion
You can change the default preferences for your profile to meet your requirements. You can set your default project, topology view (graph or list), editing medium (form or YAML), language preferences, and resource type.
```

Reply:
If "graph/list" is used in the UI, then keep it - otherwise change to "graph or list" to match style guidelines

Reply:
We have a graph view and a list view for topology. But not ”graph/list” as a single option in the UI. So, changing it as per the suggestion. Thank you!

Cc @vikram-raj